modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,90 @@
+= Create API Groups and Add API Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can bundle a set of APIs to enable ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.
+
+When you complete the task of creating an API Group with the help of this topic, you would have created an API Group, the very first version of that group, and the very first group instance. Additionally, you would also have added API instances to the API Group instance.
+
+You can add API instances to the API Group instance based on the environment (Production, Sandbox, or others) in which you create the API Group instance. For example, if you created a API Group instance in Production environment, you can add API instances to the API Group instance only from Production.
+
+You create API Groups using the following steps:
+
```

Comment:
Do we really need a list for just two tasks, and why do we not number them instead of boldfacing them?

Reply:
Numbered the tasks, but need them to clarify that you create a group in two stages.